cellxgene_schema_cli/cellxgene_schema/ontology_files/.gitignore

```diff
@@ -0,0 +1,2 @@
+*owl
+*gtf.gz
```

Comment:
add a new line at the end of the file. The `make lint` or `make fmt` commands should be catching this?

Reply:
Fixed in 1cf9ee764b2a010269bdc1f78a03699147fbbade